src/schema/rules/sidecars/dwi.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+
+
+---
+# Multipart (split) DWI schemes
+# NOTE: I don't think this can be schemafied, since it depends on owner intent.
+MRIDiffusionMultipart:
+    selectors:
+    - modality == "MRI"
+    - datatype == "dwi"
+    fields:
+        MultipartID: REQUIRED
```

Comment:
I'm not sure how we should handle MultipartID, since its requirement level depends on owner knowledge, rather than any validatable elements of the dataset.